managing-data.html

```diff
@@ -4,714 +4,912 @@
+</div>
+
+
+
+<h2>
+	<a class="anchor" id="filter"></a>
+	Filtering images
+</h2>
+
+<p>
+	You can filter images in a Dataset, to only show thumbnails that match by name or annotations.
+</p>
+
 <div class="line-block">
-<div class="line"><br /></div> <!-- end #line -->
-</div> <!-- end #line-block -->
+	<div class="line"><br /></div>
+</div>
 
-</li>
 
-<p class="top_link"><a href="#top"><img src="images/back_to_top.png" alt="Image for link to top of page" style="float: right;"/></a></p>
+<h3>
+	<a class="anchor" id="filtername"></a>
+	Filter by Name
+</h3>
 
-<h3><a class="anchor" id="rate"></a>Adding Ratings</h3>
+<ol>
+	<li>
+		<p>OMERO.insight:</p>
 
-<li><p class="first">Images can be rated from 1 to 5 in the annotations section of the <span class="bold">General</span> pane.</p>
+		<p class="indent3">- click on the <span class="bold">Filter images</span> drop-down menu to select <span class="bold">Name</span>.</p>
+		<p class="indent3">- type the text to filter by and hit Enter to filter.</p>
 
-<p>OMERO.insight:</p>
+		<p>OMERO.web:</p>
 
-<p class="indent3">- click on the stars to set your rating</p>
-<p class="indent3">- click on the <span class="bold">-</span> button to remove your rating.</p>
+		<p class="indent3">- click on the <span class="bold">Add filter</span> drop-down menu and select <span class="bold">Name</span>.</p>
+		<p class="indent3">- filtering will be performed as you enter the filter text.</p>
 
-<p>OMERO.web:</p>
-	
-<p class="indent3">- click on the <span class="bold">+</span> button</p>
-<p class="indent3">- click on the stars to set your rating</p>
-<p class="indent3">- mouse over the rating and click the <span class="bold">x</span> to remove your rating.</p>
+		<h3>OMERO.insight</h3>
+		<img alt="../images/managingFilterName.jpg" class="align-center" src="images/managingFilterName.jpg" style="width: 100%;" />
 
-<div class="line-block">
-<div class="line"><br /></div> <!-- end #line -->
-</div> <!-- end #line-block -->
 
-<h3>OMERO.insight</h3>
+		<div class="line-block">
+			<div class="line"><br /></div>
+		</div>
 
-<img alt="../images/managingRateInsight.jpg" class="align-center" src="images/managingRateInsight.jpg" style="width: 80%;" />
+		<h3>OMERO.web</h3>
+		<img alt="../images/managingFilterNameWeb.png" src="images/managingFilterNameWeb.png" style="width: 80%;" />
+
+	</li>
+</ol>
 
-<div class="line-block">
-<div class="line"><br /></div> <!-- end #line -->
-</div> <!-- end #line-block -->
 
 <div class="line-block">
-<div class="line"><br /></div> <!-- end #line -->
-</div> <!-- end #line-block -->
+	<div class="line"><br /></div>
+</div>
 
-<h3>OMERO.web</h3>
+<h3>
+	<a class="anchor" id="filtertag"></a>
+	Filter by Tag
+</h3>
 
-<img alt="../images/managingRateWeb.jpg" class="align-center" src="images/managingRateWeb.jpg" style="width: 80%;" />
+<ol>
+	<li>
+
+		<p class="first">To filter by Tag (Insight only) select <span class="bold">Tags</span> option from the filter menu.</p>
```

Comment:
Insight only is slightly lost here, I'd either put it on the heading or at the beginning of the line to match the layout of filter by rating below:
1. OMERO.insight only:
                - select Tags option from the filter menu